src/Kunstmaan/AdminBundle/composer.json

```
@@ -31,6 +31,7 @@
         "symfony/event-dispatcher": "^3.4|^4.2",
         "symfony/filesystem": "^3.4|^4.2",
         "symfony/finder": "^3.4|^4.2",
+        "symfony/mailer": "^3.4|^4.2",
```

```suggestion
        "symfony/mailer": "^4.4",
```

The mailer was introduced in symfony 4.3, but as experimental. So require the first "stable" version